docs/UserGuide.adoc

```diff
@@ -59,7 +60,7 @@ Format: `help`
 
 Format: `bye`
 
-=== CRUD
+=== Working with eateries
 ==== Adding an eatery : `add`
 
 Adds an eatery in the app +
```

Comment:
they can specify tags also but it's optional so the command for add should be like "add \n [name of eatery] \a [address] \c [category] {\t [tags, space-delimited]}"? haha idk how else to indicate that it's optional so i just used the squiggly brackets but if u have a better idea then just disregard my suggestion TT